src/reviewers/formPage1.php

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
-<?php session_start();
-include_once 'creds.php';
+<?php
```

Comment:
Should be moved all the way up before any content for `session_start` and `authorize` to work properly when output buffering is off. It will never be off, but y'know.

Reply:
this should be fine because session start is called init.php